build-tools/build-indexes

```diff
@@ -619,6 +619,8 @@ process.stdout.write("Building `data/teambuilder-tables.js`... ");
 		BattleTeambuilderTable[gen].overrideType = overrideType;
 		const overrideAbility = {};
 		BattleTeambuilderTable[gen].overrideAbility = overrideAbility;
+		const removeSecondAbility = {};
```

Comment:
This seems overkill? I believe `ability.gen == 4 && gen === 3` is the only situation you'd want to remove the second ability?

Reply:
What motivated me to do this was that from gen 6 to 7, Beartic got Slush Rush, Gigalith got Sand Stream, Pelipper got Drizzle and Vamilluxe got Snow Warning.

However, I just noticed that the gen 3->gen 4 ability gains aren't caught by this, because e.g. ``Dex.mod('gen3').data.Pokedex.hitmontop`` already has both Intimidate and Technician listed despite the latter not existing yet. Is that intentional?

Reply:
Hm that's annoying, so I will have to add the ability gen check after all. I had hoped to avoid that with this whole ordeal.